Templates/VSIX/Project Templates/cpp-winui-template/WinUI3TemplateCpp/SampleConfiguration.cpp

```diff
@@ -6,16 +6,20 @@
 #include "SampleConfiguration.h"
 #include "MainPage.xaml.h" 
 
-using namespace winrt;
-using namespace winrt::$safeprojectname$;
-using namespace Microsoft::UI::Xaml;
-using namespace Windows::Foundation::Collections;
+namespace winrt
+{
+    using namespace Microsoft::UI::Xaml;
+    using namespace Windows::Foundation::Collections;
+}
 
-IVector<Scenario> implementation::MainPage::scenariosInner = winrt::single_threaded_observable_vector<Scenario>(
-	{
-		Scenario{ L"Scenario 1", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario1_ShortName>())},
-		Scenario{ L"Scenario 2", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario2_ShortName>())}
-	});
+namespace winrt::$safeprojectname$
+{
+    IVector<Scenario> implementation::MainPage::scenariosInner = winrt::single_threaded_observable_vector<Scenario>(
+        {
+            Scenario{ L"Scenario 1", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario1_ShortName>())},
```

Comment:
Now that you're inside the `winrt` namespace (per line 15) you don't need the `winrt` prefixes inside it.  Name resolution will find them. So:

* `winrt::single_threaded_observable_vector` -> `single_threaded_observable_vector`
* `winrt::hstring` -> `hstring` (if still necessary)
* `winrt::name_of<>` -> `name_of<>`